package.json

```diff
@@ -11,7 +11,8 @@
     "test": "test",
     "check": "npm run lint && prettier --check .",
     "lint": "eslint --fix ./src ./test",
-    "format": "prettier --write ."
+    "format": "prettier --write .",
```

Comment:
prettier and formatting did not work on my local

Reply:
Hmmm interesting, it works for me. I just updated the formatting: https://github.com/commercetools/commercetools-subscriptions/pull/2/commits/64f5ea5c0ba67bd483b381a23c3dd19e9e239899

If it doesn't work for you, let's wait until we get CI set up, and then we can check.